core/templates/dev/head/domain/objects/objects-domain.constants.ts

```diff
@@ -18,6 +18,8 @@
 
 export class ObjectsDomainConstants {
   public static FRACTION_PARSING_ERRORS = {
+    INVALID_CHARS_LENGTH:
+      'None of the numbers of the fraction should be larger than 7 digits.',
```

Comment:
@aks681 This is the exact string that's present in the android version. Changing this might disrupt the parity between android and web. Should I still make the change?

Reply:
Oh, ok. Fine then!

Reply:
Actually, @aks681's version of the error message is better. Can you please fix it here and also file an issue on oppia/oppia-android to update the error message?

Reply:
Done and file an [issue](https://github.com/oppia/oppia-android/issues/672).